.kokoro/populatesecrets.sh

```diff
@@ -7,11 +7,27 @@ declare -r SECRETS_LOCATION="${KOKORO_GFILE_DIR}/secret_manager"
 populate_all_secrets(){
     echo "Creating folder on disk for secrets: ${SECRETS_LOCATION}"
     mkdir -p ${SECRETS_LOCATION}
-    for key in $(echo ${COMMON_SECRETS} | sed "s/,/ /g")
+
+    populate_secrets "${COMMON_SECRETS}" "cloud-sharp-jenkins"
```

Comment:
Does calling a function declared later work? I thought bash needed the function come before any calls to it. I suspect you've tested this and it works... I'm just surprised.

Reply:
It does work, although it's true that shell has no forward declaration. But I think this works because I'm calling populate_secrets from another function (populate_all_secrets) and not from standalone code at the start of the file, that is by the time populate_all_secrets is actually called, this whole file has been "executed" which means both functions have been declared.